structs/ctl/message.go

```diff
@@ -70,6 +70,10 @@ func requestType(req interface{}) (string, error) {
 		return CommandAddFile, nil
 	case GetFileRequest, *GetFileRequest, GetFileResponse, *GetFileResponse:
 		return CommandGetFile, nil
+	case DataReadRequest, *DataReadRequest:
```

Comment:
We should probably add the Response types to these cases as well, even if they are unused. More consistent + they will be there if we ever decide to use them